docs/qprofiler2/index.md

```diff
@@ -306,7 +306,7 @@ In above bam summary output example, this metrics lists the summed counts from a
 
 
 ### BAM metrics output
-This <bamMetrics> element lists static data based on BAM record fields order, including all mandatory fields and some optional fields in the SAM format. In each BAM record field, qprofiler2 takes different read counts. eg. it only counts mapped records to summary the POS field, but counts all records to summary the FLAG field. The details lists below:
+The static data whithin the element <bamMetrics> is based on BAM record fields order, including all mandatory fields and some optional fields in the SAM format. The qprofiler2 takes different read count in each BAM record field. For example, only the mapped records are counted towards the POS field, whereas the FLAG field counts all records. The details lists below:
```

Comment:
How about:

The static data within the element <bamMetrics> is based on the order of the BAM record fields, and includes all mandatory fields and some optional fields. qprofiler2 will update different fields depending on the BAM record. For example, only the mapped records count towards the POS field tally, whereas all records will be counted in the FLAG field. Details are listed below:

Reply:
fixed, please review